web-api/openapi/schemas/id.yml

```diff
@@ -2,7 +2,7 @@ Id:
   description: |-
     An id can be any non-empty string that consist of digits, latin letters, underscores, colons, minus signs, at signs, and dots.
   type: string
-  pattern: '^[a-zA-Z0-9_:@.-]+$'
+  pattern: '^[a-zA-Z0-9_:@.\-]{1,256}$'
   minLength: 1
   maxLength: 256
```

Comment:
This now duplicates the length constraints making them more prone to change induced bugs/desync.

Reply:
but with just the `+` repetition, the pattern contradicts the limits. so how should we formulate this instead? i can also revert this and we can find a solution in a follow up, but this conflict should be resolved.

Reply:
I would see it as the length further restricting the generic pattern in a similar way you would have "english grammar" and text size limitations of a book separate.

Reply:
mhh ok, if the constraints are intersected, than it's sufficient 👌